src/api-reference/user-provisioning/spend/v4.spend-user-retrieval.md

```diff
@@ -36,82 +55,83 @@ layout: reference
 ```shell
 GET https://www.us.api.concursolutions.com/spend/v4/Users
 ```
+
 #### Parameters
 
-|Name|Type|Format|Description|
-|---|---|---|---|
-|`startIndex`|`integer`|-|The starting index of the paginated result (indexes from 1).|
-|`itemsPerPage`|`integer`|-|The number of user resources to return on a single page. Maximum: 100|
-|`filter`|`string`|-|The SCIM compliant filter string to be used when retrieving user resources.|
+| Name           | Type      | Format | Description                                                                 |
+| -------------- | --------- | ------ | --------------------------------------------------------------------------- |
+| `startIndex`   | `integer` | -      | The starting index of the paginated result (indexes from 1).                |
+| `itemsPerPage` | `integer` | -      | The number of user resources to return on a single page. Maximum: 100       |
+| `filter`       | `string`  | -      | The SCIM compliant filter string to be used when retrieving user resources. |
```

Comment:
Do we have a sample request for this? Maybe we can mention it needs to be in `ABNF` syntax?